scripts/templates/EmptyWebpackConfig.mustache

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const resources = require('../../scripts/tasks/webpack-resources');
+
+const BUNDLE_NAME = '{{packageName}}';
+const IS_PRODUCTION = process.argv.indexOf('--production') > -1;
+
+module.exports = resources.createConfig(BUNDLE_NAME, IS_PRODUCTION, {
+  entry: {
+    [BUNDLE_NAME]: './lib/index.js'
```

Comment:
I think this was the cause of the release build failure this morning...

Reply:
Added index.ts and version.ts to the create-package output.